BP/common/landed_titles/BP_empire_roman.txt

```diff
@@ -32778,195 +33587,401 @@ e_britannia = {
+					old_frisian = cn_arbeidsmerk
+					#Romance Cultures
+					norman = cn_glamorgan
+					#Germanic Cultures
+					english = cn_glamorgan
 				}
-
 				b_casnewydd = {	#Caerllion
 					province = 1630
-
-					color = { 210 165 60 }
-					color2 = { 255 255 255 }
-
 					cultural_names = {
+						#Romance Cultures
 						norman = cn_caerleon
+						#Germanic Cultures
 						english = cn_caerleon
 					}
+					color = { 210 165 60 }
+					color2 = { 255 255 255 }
 				}
 
 				b_monmouth = {	#Trefynwy
 					province = 1631
-
-					color = { 210 165 60 }
-					color2 = { 255 255 255 }
-
 					cultural_names = {
+						#Roman Cultures
+						roman = cn_blestium
+						romano_gallic = cn_blestium
+						romano_hispanic = cn_blestium
+						romano_british = cn_blestium
+						african_romance = cn_blestium
+						raetoroman = cn_blestium
+						thraco_roman = cn_blestium
+						illyro_roman = cn_blestium
+						pannonian = cn_blestium
+						dalmatian = cn_blestium
+						noricroman = cn_blestium
+						romano_gothic = cn_blestium
+						romano_dacian = cn_blestium
+						romano_frank = cn_blestium
+						romano_aramean = cn_blestium
+						romano_vandalic = cn_blestium
+						#Celtic Cultures 
+						briton = cn_abermynwy
+						#Central Germanic Cultures
+						anglo = cn_monnowmuda
+						old_saxon = cn_monnowmuda
+						#Romance Cultures
 						norman = cn_monmouth
+						#Germanic Cultures
 						english = cn_monmouth
 					}
+					color = { 210 165 60 }
+					color2 = { 255 255 255 }
 				}
 			}
 			c_brecknockshire = { #Brycheiniog"
 				color = { 215 150 115 }
 				color2 = { 255 255 255 }
-
 				cultural_names = {
+					#Romance Cultures
 					norman = cn_brecknockshire
+					#Germanic Cultures
 					english = cn_brecknockshire
 				}
-
 				b_brecon = {	#Aberhonddu
 					province = 1659
-
-					color = { 215 150 115 }
-					color2 = { 255 255 255 }
-
 					cultural_names = {
-						#Brecon is the modern form.
+						#Roman Cultures
+						roman = cn_cicucium
+						romano_gallic = cn_cicucium
+						romano_hispanic = cn_cicucium
+						romano_british = cn_cicucium
+						african_romance = cn_cicucium
+						raetoroman = cn_cicucium
+						thraco_roman = cn_cicucium
+						illyro_roman = cn_cicucium
+						pannonian = cn_cicucium
+						dalmatian = cn_cicucium
+						noricroman = cn_cicucium
+						romano_gothic = cn_cicucium
+						romano_dacian = cn_cicucium
+						romano_frank = cn_cicucium
+						romano_aramean = cn_cicucium
+						romano_vandalic = cn_cicucium
+						#Celtic Cultures 
+						briton = cn_aberhonddu
+						#Central Germanic Cultures
+						anglo = cn_brychan
+						old_saxon = cn_brychan
+						#Romance Cultures
 						norman = cn_brecknock
+						#Germanic Cultures
 						english = cn_brecknock
 					}
+					color = { 215 150 115 }
+					color2 = { 255 255 255 }
 				}
-				
-				b_talgarth = {
+				b_talgarth = {  #Talgarth is actually the modern & historic name in both English & Welsh, albeit accented in Old Welsh.
```

Comment:
We will have to look at those accentys then

Reply:
Alright I'll take a gander. A quick review throughout Wikipedia doesn't give me any results, although it does say: "It appears as Talgart in 1121, as Talgard after 1130 and in its present form between 1203-1208". We could possibly use cn_talgart instead? Let me know what you think.